src/components/Selector/MultiSelector.tsx

```diff
@@ -66,7 +66,7 @@ export function MultiSelector<
           aria-invalid={!!get(errors, name)?.message}
           aria-disabled={isDisabled}
           as="div"
-          className={`${button} ${styles.selectorButton} p-1 focus-within:border-gray-d1 focus-within:dark:border-blue-l2`}
```

Comment:
bug: tabbing into `<Multiselector/>` doesn't focus
repro: 
go to registration contact details and tab from website to sdg
<img width="804" alt="image" src="https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/89639563/e18e5968-9fe3-47fa-af8c-f88aca2ea7a2">

Reply:
Fixed https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2568/commits/72381756f65dbb2cff34f58cd45960eb88baad71